tap_kit/client.py

```diff
@@ -1,3 +1,5 @@
+from builtins import Exception
```

Comment:
No need to import this, Python knows about `Exception` (see the base class of `RateLimitException`)